src/modules/nft-manager/handlers/handlerNftManager.js

```
@@ -16,18 +19,70 @@ export default class NFT {
-      while (result.next) {
-        const res = await fetch(result.next).then(response => response.json());
-        result = res.result;
-        nftResults = nftResults.concat(result.nfts);
+      const { data, error } = await fetch(endpoint).then(response =>
+        response.json()
+      );
+      if (error) {
+        return null;
       }
-      return nftResults;
+      const items = data && data.items ? data.items : [];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)